locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -1065,6 +1066,13 @@ protected MachineLocation obtainOnce(ConfigBag setup) throws NoMachinesAvailable
                 LOG.warn("Problem generating log message summarising completion of jclouds machine provisioning "+machineLocation+" by "+this, e);
             }
 
+            if (customizers.size() > 0) {
+                machineLocation.config().set(JCLOUDS_LOCATION_CUSTOMIZERS, customizers);
+            }
+            if (machineCustomizers.size() > 0) {
+                machineLocation.config().set(MACHINE_LOCATION_CUSTOMIZERS, machineCustomizers);
+            }
```

Comment:
Can you instead add the config for `JCLOUDS_LOCATION_CUSTOMIZERS` at [construction time](https://github.com/apache/brooklyn-server/blob/master/locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java#L2056-L2099) so migrated settings are all at one place.